plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -57,6 +57,7 @@
 import org.apache.parquet.io.MessageColumnIO;
 import org.apache.parquet.schema.GroupType;
 import org.apache.parquet.schema.MessageType;
```

Comment:
Shorten commit message to
`Implement predicate pushdown for ROW sub fields in parquet for hive`

Reply:
done.